### DISCUSSION TITLE
Missing FET tokens and claimed rewards on my Fetch wallet

### DISCUSSION THREAD

**Discussion by AlohaCrypto808** [community/community#48030] [Other Feature Feedback, Questions, & Ideas]
Has anyone else had this happen?
I sent FET like i always do from Kraken to my Fetch wallet. They never arrived. Tonight I claimed my rewards so that i could restake them and they never arrived. I cant find a Fetch wallet support or place to view what happened.

**Comment by queenofcorgis** [comment]
Hi @AlohaCrypto808,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Fetch Wallet. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
